# dimensions of the map
float32 width
float32 height
float32 margin

# resolution of the thetaStar map
float32 resolutionThetaStar

# Info about the trays
Rectangle[] obstacles